leios-trace-verifier/Parser.agda

```diff
@@ -70,35 +83,118 @@ record TraceEvent : Type where
 {-# COMPILE GHC TraceEvent = data TraceEvent (TraceEvent) #-}
 
 open import Leios.SpecStructure using (SpecStructure)
-open import Leios.Trace.Verifier
 
-open import Leios.Defaults 2 fzero using (st)
+open import Leios.Defaults 10 fzero hiding (LeiosInput; LeiosOutput; SLOT)
+open import Leios.Trace.Verifier 10 fzero
 open import Leios.Short st
+import Data.String as S
+
+nodeId : String → Fin 10
+nodeId "node-0" = fzero
```

Comment:
I think you can write `# 0`, etc.

Reply:
As the number of nodes (here 10) is now read from the configuration file, this function has become more generic as well